instrumentation/opentelemetry-instrumentation-kafka-python/src/opentelemetry/instrumentation/kafka/package.py

```diff
@@ -13,4 +13,5 @@
 # limitations under the License.
 
 
-_instruments = ("kafka-python >= 2.0",)
+_instruments = ("kafka-python >= 2.0, < 3.0",)
```

Comment:
Could you please keep _instruments unique and handle it in something like this?
https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2702/files#diff-616b2ef8de6c8fe9cc6278d0adaadb35a4a81667400d476ff472373049b6b20aR16

Reply:
Updated